lib/_debug_agent.js

```diff
@@ -21,7 +21,7 @@ exports.start = function start() {
   agent.listen(process._debugAPI.port, function() {
     var addr = this.address();
     process._rawDebug('Debugger listening on port %d', addr.port);
-    process._debugAPI.notifyListen();
+    process._debugAPI.notifyListen(addr.port);
```

Comment:
so could it be possible that an error was passed to the callback from listen and `this.address().port` is undefined? If so, that would cause an abort in debug-agent right? Should we guard for that?